_data/experience2.yml

```diff
@@ -0,0 +1,19 @@
+- role: Senoir Code Reviewer
+  company: Microverse
+  dates: May 2021 - Present
+  logo: fellowship.svg
+
```

Comment:
You can still use the Microverse logo here
![Microverse](https://media-exp1.licdn.com/dms/image/C560BAQHr8P7gQ95yCQ/company-logo_200_200/0/1578673850004?e=1635379200&v=beta&t=lcOlWKQuwFd5bpmDh3NJkZmNZXD7QC6KE8SKuVZJ0ok)